lib/oci_utils/impl/oci-image-migrate-main.py

```diff
@@ -158,13 +158,13 @@ def show_utilities(found, missing):
         No return value.
     """
     if found:
-        print('\n  %30s\n%s' % ('  Utilities found:', '  ' + '-'*60))
+        print(('\n  %30s\n%s' % ('  Utilities found:', '  ' + '-'*60)))
```

Comment:
double brackets ....

Reply:
ok.
The code should not have been  python3 at the start .. 
I'll try to parse the code to check such double paranthesis

Reply:
the oci-image-migrate code is full python3 since early Jan; just get the .py files from the master or from my branch